docs/troubleshooting.rst

```diff
@@ -82,3 +82,22 @@ can also happen with other database encoding. The encoding expected by kinto is
 To remediate this, you can issue the following command, once ``pgsql`` open::
 
   update pg_database set encoding = pg_char_to_encoding('UTF8') where datname = '<your db name>';
+
+
+bind: address already in use
+============================
+
+You will probably have a more precise error message telling you which
+port is already in use: ``listen tcp 0.0.0.0:5432: bind: address
+already in use``.
+
+This happens when you are trying to start a docker image on the same
+port of an existing service running on your machine.
```

Comment:
This is already describe earlier: https://github.com/Kinto/kinto/pull/637/files/cd2788b4649f3a520c1c0e105e640442c174ac5c#diff-4f31c1e2c9e43e4d6edbcd36ed58f8b8R38